### ISSUE TITLE
[Research]: Inventory management adding details

### ISSUE THREAD

**Issue by Dianadec** [bcgov/reserve-rec-public#62]
#### Description:

Discuss with developers, reservation services team, etc.

Need to:
- opportunities where there are overlap, deviations, etc
- Building on policy work that Nicole did.

#### Acceptance Criteria:
- [ ] book time with reservation services: is everything we have what we need, what else can we add, what can we simplify, etc.
- [ ] Bonus? - consider looking at New Zealand and Washington State

#### Dependencies

- Blocked by
- Blocking

**Relevant documentation as reference**

**Definition of Ready**

- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if applicable)
- [ ] Design / Solution is accepted by Product Owner (if applicable)
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** 
- In progress:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] UI meets accessibility requirements
  - [ ] Unit tests are written
  - [ ] Work is traceable in GitHub
  - [ ] PR linked to ticket number
  - [ ] If needed/required - Dev adds flag/label to highlight any migration steps necessary prior to PROD deployment
- Code review:
  - [ ] Code is peer reviewed and has passed CI/CD tests
- QA:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Code is potentially shippable to the production environment
  - [ ] Functional features have been tested and passed by QA
  - [ ] UI components tested by designer
  - [ ] Code is deployed to PROD when moved to 'done' column (unless requested otherwise by PO)
- PO Review:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Reviewed and approved by Product Owner

#### Notes:
- 
-

**Comment by Dianadec** [comment]
For this ticket, @pavelcreates Daniel suggests to get up to date with the current data model piece - suggest connecting with @cameronpettit  and @danieltruong .

Cc @LindsayMacfarlane

**Comment by Dianadec** [comment]
Standup notes:
next steps: 
- identify which form elements are consistent through each type
- do an exercise with reservation services to explore limitations from current provider, etc

**Comment by Dianadec** [comment]
In future - look into booking windows and streamlining them.

**Comment by pavelcreates** [comment]
Could I get some further clarification on the expectations for the need of "Building on policy work that Nicole did."? 

The vision I have for these workshops would likely lead to participants organically bringing up those policies for further discussion, but not dive into them in greater detail at the time of the workshops. The focus being the forms that users are filling out, specifically. What may work best for this ticket would be "Identify policies for future discussion with program area" or something like that.

Naturally, we will get into policy-related topics, but I would suggest that if we wanted to get more in depth with those policies, to create a separate ticket for a more detailed exploration of the policies as a follow-up to this ticket.

Does that align with what you had in mind? @Dianadec @LindsayMacfarlane

**Comment by LindsayMacfarlane** [comment]
I feel you've captured the 'policy' piece a bit within your existing work/field mapping in Mural. I could be mistaken, my thought around that bullet was primarily to see what work has already been done by Nicole in terms of mapping out information. One thing I'd like us to start doing is understanding where and what policies might limit implementation and where there may may be opportunities to work with the reservation services team to update policies.

**Comment by Dianadec** [comment]
Workshop happening on January 22nd. The Card sorting activity is open and in progress until the workshop.

**Comment by Dianadec** [comment]
Todo: create synthesis ticket for this work.

**Comment by LindsayMacfarlane** [comment]
Before closing this ticket, @pavelcreates / @Dianadec - could we please add in links to materials.
